packages/replay/worker/src/Compressor.ts

```diff
@@ -25,20 +25,21 @@ export class Compressor {
     return;
   }
 
-  public addEvent(data: Record<string, unknown>): void {
+  public addEvent(data: Record<string, unknown>): boolean {
     if (!data) {
-      return;
+      return false;
```

Comment:
If we just throw an error here, it would result in the response:

```js
 postMessage({
        id,
        method,
        success: false,
        response: err,
      });
```

And a rejected promise, which we could handle. Wouldn't that be more straightforward? Then we could keep everything as `Promise<void>` in the actual replay code itself (as we don't really care about the response, and it is also inconsistent between worker/non-worker compression)

Reply:
Ah yeah we could do that -- at one point, I was returning data, but decided to simplify it a bit for now.